membership_extension/models/membership_category.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class MembershipCategory(models.Model):
+    _name = "membership.membership_category"
+    _description = "Membership category"
+
+    name = fields.Char(required=True, translate=True)
```

Comment:
This is the same use case of ```res.partner.category``` where we have a many2many field related with partner.
We can define two, one to partners and one to product.template.
But I think we should think about the views and queries before create two more tables in DB, and then this could be an improvement in the future